src/dataflow/graph.h

```diff
@@ -228,8 +228,6 @@ struct Graph : public UnifiedExpressionVisitor<Graph, Node*> {
       return doVisitUnreachable(unreachable);
     } else if (auto* drop = curr->dynCast<Drop>()) {
       return doVisitDrop(drop);
-    } else if (curr->is<Try>() || curr->is<Throw>() || curr->is<Rethrow>()) {
-      Fatal() << "DataFlow does not support EH instructions yet";
```

Comment:
Should we not keep this code? DataFlow will not work on EH until we implement something custom here. I'm not sure what would happen if we fall through to the generic path after this, but a clear error seems fine.

Reply:
I think you're right; I just mechanically reverted the previous PR because I thought now we are OK with Flatten, but yeah I think we should keep this. Thanks.